swift/ql/src/queries/Security/CWE-312/CleartextLogging.qhelp

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+Sensitive information that is logged unencrypted is accessible to an attacker
+who gains access to the logs.
+</p>
+</overview>
+
+<recommendation>
+<p>
+Ensure that sensitive information is always encrypted or obfuscated before being
+logged.
+</p>
+
+<p>
+In general, decrypt sensitive information only at the point where it is
+necessary for it to be used in cleartext.
+</p>
+
+<p>
+Be aware that external processes often store the standard out and
+standard error streams of the application, causing logged sensitive
+information to be stored.
+</p>
+</recommendation>
+
+<example>
+<p>
+The following example code logs user credentials (in this case, their password)
+in plain text:
```

Comment:
Are we using 'plain text' and 'cleartext' as synonyms in the text in this diff? If so perhaps use one or the other? 

If not and this is intentional, feel free to ignore and resolve this comment!

Reply:
Sorry, one further minor comment: seems we should use `plaintext` per the MS style guide: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/plaintext-plain-text

```suggestion
in plaintext:
```